### ISSUE TITLE
[Bug]: Skeleton aria-label is hardcoded in English

### ISSUE THREAD

**Issue by dmytrokirpa** [microsoft/fluentui#31961]
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
System:
    OS: macOS 14.5
    CPU: (16) arm64 Apple M3 Max
    Memory: 430.70 MB / 64.00 GB
    Shell: 5.9 - /bin/zsh
  Browsers:
    Edge: 126.0.2592.87
    Safari: 17.5
```

### Are you reporting Accessibility issue?

None

### Reproduction

https://codesandbox.io/p/sandbox/skeleton-aria-label-repro-forked-gdysx9

## Actual Behavior
The [default `aria-label`](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-skeleton/library/src/components/Skeleton/useSkeleton.ts#L27) for the Skeleton component should not be hardcoded in English.

## Expected Behavior
If an aria-label is necessary for accessibility, it should be a required prop for the component.

_Originally posted in https://github.com/microsoft/fluentui/issues/31193#issuecomment-2102127947_

### Requested priority

Normal

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by khmakoto** [comment]
After confirming with the team, we have agreed to remove the default `aria-label`. @tomi-msft will take care of that.
